test/usage.test.ts

```diff
@@ -401,3 +401,15 @@ test('Another limitation: passing `this` references to `expectTypeOf` results in
   // Instead of the above, try something like this:
   expectTypeOf(B).instance.toEqualTypeOf<{b: string; foo: () => void}>()
 })
+
+test('Use `.pick` to pick a set of properties from an object', () => {
+  type Person = {name: string; age: number}
+
+  expectTypeOf<Person>().pick<'name'>().toEqualTypeOf<{name: string}>()
+})
+
+test('Use `.omit` to remove a set of properties from an object', () => {
+  type Person = {name: string; age: number}
+
+  expectTypeOf<Person>().omit<'name'>().toEqualTypeOf<{age: number}>()
+})
```

Comment:
Could you move these up to come after the extract/exclude tests? And then run `pnpm lint --fix` - that will update the readme too.

Reply:
Sure thing, I will do that!